tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -108,6 +113,11 @@ IncludeByKey.NUM_SDW_AMP_LINKS {
 					num_input_audio_formats 1
 					num_output_audio_formats 1
 					num_output_pins 1
+IncludeByKey.SDW_AMP_FORMAT {
+"[s*le]"	{
```

Comment:
@bardliao I dont think you need the IncludeByKey at all. The format attribute is already resitricted to the s*le and float types

Reply:
> @bardliao I dont think you need the IncludeByKey at all. The format attribute is already resitricted to the s*le and float types

I added it because otherwise we will set format = '' by default which is invalid.